sqlitecluster/SQLiteNode.cpp

```diff
@@ -686,9 +686,9 @@ bool SQLiteNode::update() {
                         numFullResponded++;
                         numFullApproved += SIEquals(response, "approve");
                         if (!SIEquals(response, "approve")) {
-                            SWARN("Peer '" << peer->name << "' declined transaction.");
+                            SWARN("Peer '" << peer->name << "' denied transaction.");
```

Comment:
Yeah, that's reasonable. If it fails on any slave we're not going to be able to safely get it onto the whole cluster.